### PR TITLE
Move deps to git submodule

### PR DESCRIPTION
**Pull request by qihqi**
* Make deps into submodule - the hash of particular submodule is stored in git's database
* Made run_interactive.py with tiny to a CI
* fix some issues with run_interactive

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[submodule "deps/JetStream"]
```
Comment (FanhaiLu1) [inline .gitmodules:1]: Does it mean the default is head? In automation test or integration test, we would need the tag or commit id for the project.

Comment (qihqi) [reply .gitmodules:1]: the default is this hash: 
image

To update this hash one ch's into the directory and run `git pull ..` etc.

File: run_interactive.py [hunk 2]
```diff
@@ -158,20 +158,15 @@ def main(argv):
       decode_state, result_tokens = engine.generate(params, decode_state)
       result_tokens = result_tokens.convert_to_numpy()
       res = result_tokens.get_result_at_slot(slot)
-      stop_tokens = set(tokenizer.tokenizer.stop_tokens)
+      stop_tokens = set(tokenizer.stop_tokens)
```
Comment (wang2yn84) [inline run_interactive.py:161]: JetStream didn't implement the stop_tokens for SentencePiece. You might need to add that function to JetStream.

File: run_interactive.py [hunk 3]
```diff
@@ -158,20 +158,15 @@ def main(argv):
...
-      token_id = res.tokens[0][0]
-      sampled_tokens_list.append(token_id)
-      # output_str = tokenizer.decode_str([token_id])
-      # print(Fore.GREEN + output_str, end="", flush=True)
-
-    # print(Style.RESET_ALL + "\n")
-    # print("---- Streaming decode finished.")
 
     print("---- All output tokens.")
     print(sampled_tokens_list)
```
Comment (wang2yn84) [inline run_interactive.py:172]: I tried in my local, actually the decode function doesn't work. Not sure if it's my local env or not.

File: run_interactive.py (hunk 2 above)
Comment (qihqi) [reply run_interactive.py:161]: I did: https://github.com/google/JetStream/pull/84

**Review by FanhaiLu1 — APPROVED** [review]

**Pull request merged**